debug/lib.h

```diff
@@ -28,19 +28,52 @@
  * | File                | Description                |
  * | :------------------ | :------------------------- |
  * | debug/backtrace.c   | @subpage debug_backtrace   |
+ * | debug/graphviz.c    | @subpage debug_graphviz    |
+ * | debug/notify.c      | @subpage debug_notify      |
  * | debug/parse_test.c  | @subpage debug_parse       |
+ * | debug/window.c      | @subpage debug_window      |
  */
 
 #ifndef MUTT_DEBUG_LIB_H
 #define MUTT_DEBUG_LIB_H
 
 #include "config.h"
+#include "mutt/lib.h"
+#include "core/lib.h"
 
-#ifdef HAVE_LIBUNWIND
+struct MuttWindow;
+
+// Backtrace
 void show_backtrace(void);
-#endif
+
+// Graphviz
+void dump_graphviz(const char *title);
+
+// Notify
+int         debug_notify_observer(struct NotifyCallback *nc);
+const char *get_config_type      (int id);
+const char *get_context          (int id);
+const char *get_event_type       (enum NotifyType type);
+const char *get_global_event     (int id);
+const char *get_mailbox_event    (int id);
+const char *get_mailbox_type     (enum MailboxType type);
+void        notify_dump_account  (struct NotifyCallback *nc);
+void        notify_dump_color    (struct NotifyCallback *nc);
+void        notify_dump_command  (struct NotifyCallback *nc);
+void        notify_dump_config   (struct NotifyCallback *nc);
+void        notify_dump_context  (struct NotifyCallback *nc);
+void        notify_dump_email    (struct NotifyCallback *nc);
+void        notify_dump_global   (struct NotifyCallback *nc);
+void        notify_dump_mailbox  (struct NotifyCallback *nc);
```

Comment:
Don't you want to split these into dedicated headers, e.g. debug/notify.h, and guard the inclusion?

Reply:
I've stripped them out.  I'll put them in a separate file if they're needed individually.